core/naive-strategy.go

```diff
@@ -131,6 +131,8 @@ func (st *NaiveStrategy) UnrelayedPackets(src, dst *ProvableChain, sh SyncHeader
 		return nil, err
 	}
 
+	defer logger.TimeTrack(now, "UnrelayedPackets", "num_src", len(srcPackets), "num_dst", len(dstPackets))
```

Comment:
@dongrie Is it intentional to exclude L143-174 from the time tracking?

Reply:
I missed the includeRelayedButUnfinalized part.
https://github.com/hyperledger-labs/yui-relayer/pull/117/commits/9661a8d839c78e4a3ef80c58b4f002653df4dece
Fixed

Reply:
@dongrie I think we can omit `defer` in this case?

Reply:
The process is the same with or without defer now, but when the process is added after line 134 in the future, there is no need to worry about logger.TimeTrack, so I use defer.

Reply:
@dongrie I agree with you. But didn't you need to move the logging to after the `includeRelayedButUnfinalized part`? Anyway, I can approve this PR. Thank you!